packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -132,8 +134,16 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const adapter = useAdapter();
   const theme = useTheme();
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
-  const [inlineImageUploads, handleInlineImageUploadAction, onUploadInlineImage, onCancelInlineImageUploadHandler] =
-    useImageUpload();
+  const [
+    editBoxImageUploadsInProgress,
```

Comment:
Do we really need to have these functions as part of the hook, instead of having them as separate functions?

Reply:
Those are all state related functions so it has to be in the hook I believe.

Reply:
> Can we send the states as parameters instead?

I feel that's a bit anti-pattern for how hooks work? If the concern is having too many export/imports, maybe we can group them under editBoxHook and sendBoxHook and use them as editBoxHook.handleEditBoxInlineImageUploadAction?

Reply:
Sorry, I didn't mean to add parameters as part of the hook, but I meant instead of having useImageUpload hook that returns a bunch of functions, to have separate functions that take adapter, inlineImageUploads and handleInlineImageUploadAction (probably something else) as parameters.

I'm a little bit worried that the retuned values list for this hook is pretty big and it may be confusing later